src/modules/users/user.entity.ts

```diff
@@ -69,4 +69,108 @@ export class User {
   })
   @Column({ nullable: true, default: null })
   googleId: string | null;
+
+  @ApiProperty({
+    example: true,
+    description: 'Indicates if the users profile is active',
+  })
+  @Column({ default: true })
+  isAccountActive: boolean;
+
+  @ApiProperty({
+    example: 'file-1718301871158-882823500.jpg',
+    description: 'The users profile photo',
+  })
+  @Column({
+    nullable: false,
+    default: './uploads/avatars/file-1718301871158-882823500.jpg',
+  })
+  photoUrl: string;
+
+  @ApiProperty({
+    example: '+1234567890',
+    description: 'Phone number of the user',
+  })
+  @Column({ nullable: true })
+  phoneNumber: string;
+
+  @ApiProperty({
+    example: '123 Main St',
+    description: 'Address line 1 of the user',
+  })
+  @Column({ nullable: true })
+  addressLine1: string;
+
+  @ApiProperty({
+    example: 'Apt 101',
+    description: 'Address line 2 of the user',
+  })
+  @Column({ nullable: true })
+  addressLine2: string;
+
+  @ApiProperty({
+    example: 'Canada',
+    description: 'Country of the user',
+  })
+  @Column({
+    nullable: false,
+    default: 'Canada',
+  })
+  country: string;
+
+  @ApiProperty({
+    example: 'California',
+    description: 'State of the user',
+  })
+  @Column({ nullable: true })
+  state: string;
+
+  @ApiProperty({
+    example: 'Los Angeles',
+    description: 'City of the user',
+  })
+  @Column({ nullable: true })
+  city: string;
+
+  @ApiProperty({
+    example: 'L',
+    description: 'Clothes size of the user',
+  })
+  @Column({ nullable: true })
+  clothesSize: string;
+
+  @ApiProperty({
+    example: 'W 27 H 33',
+    description: 'Jeans size of the user',
+  })
+  @Column({ nullable: true })
+  jeansSize: string;
+
+  @ApiProperty({
+    example: '10',
+    description: 'Shoes size of the user',
+  })
+  @Column({ nullable: true })
+  shoesSize: string;
+
+  @ApiProperty({
+    example: '1234 5678 9012 3456',
+    description: 'Card number of the user',
+  })
+  @Column({ nullable: true })
+  cardNumber: string;
+
+  @ApiProperty({
+    example: '12/24',
+    description: 'Expiration date of the card',
+  })
+  @Column({ nullable: true })
+  expireDate: string;
+
```

Comment:
Are we going to store all the user's card details, including the CVV, in the website's database? Is this even legal? I believe a PCI DSS certificate is required for this. Will the client have it?